sections/advanced/email-testing.md

```diff
@@ -213,3 +215,80 @@ cy.task('findEmailToUser', recipientEmail).then(emailContent => {
   // https://github.com/muratkeremozcan/cypressExamples/blob/master/cypress-api-testing/cypress/integration/firstTest.spec.js
 });
 ```
+
+## (4) The overhead is abstracted away with [Cypress Mailosaur plugin](https://www.npmjs.com/package/cypress-mailosaur)
+
+Mailosaur team released a Cypress plugin in mid 2020. With it, we do not have to replicate any complex API utities or use cy.task via Mailosaur npm package; none of what you have seen in section (3) is necessary. There is no need to create cy.task utilities or even hyberdize them. With the Cypress Mailosaur plugin, you can just use the custom Cypress commands Mailosaur team created for us.
+
+All we need is to install the package `npm install cypress-mailosaur --save-dev` and add the following line to cypress/support/index.js:
+`import 'cypress-mailosaur'`.
+
+Mailsaur plugin has a few handy functions which help you abtract complex needs.
+A full list can be found at at https://github.com/mailosaur/cypress-mailosaur
+
+```
+mailosaurListServers()
+mailosaurCreateServer({ name })
+mailosaurGetServer(serverId)
+mailosaurUpdateServer(serverId, server)
+mailosaurDeleteServer(serverId)
+mailosaurListMessages(serverId)
+mailosaurCreateMessage(serverId)
+mailosaurGetMessage(serverId, criteria)
+mailosaurGetMessageById(messageId)
+mailosaurSearchMessages(serverId, criteria, options)
+mailosaurGetMessagesBySubject(serverId, subjectSearchText)
+mailosaurGetMessagesByBody(serverId, bodySearchText)
+mailosaurGetMessagesBySentTo(serverId, emailAddress)
+mailosaurDeleteMessage(messageId)
+mailosaurDeleteAllMessages(serverId)
+mailosaurDownloadAttachment(attachmentId)
+mailosaurDownloadMessage(messageId)
+mailosaurGetSpamAnalysis(messageId)
```

Comment:
Is the API list necessary? It could become obsolete in a while with next plugin updates but please, let me know what do you think 😉

Reply:
you are right, too much information is distracting. It is already at the link.